source/frontend/src/features/properties/map/MapContainer.test.tsx

```diff
@@ -262,13 +270,7 @@ describe('MapContainer', () => {
     expect(basemapToggle).toHaveAttribute('src', '/streets.jpg');
   });
 
-  it('shows the current map scale', async () => {
-    await setup();
-    expect(document.querySelector('.leaflet-control-scale')).toBeVisible();
-    expect(document.querySelector('.leaflet-control-scale-line')).toHaveTextContent(/100 km/i);
-  });
-
```

Comment:
Why did we remove this test? was it moved to another file? I think it is asserting that the new scale widget is present and showing proper information

Reply:
merge issue, will correct.